packages/appstore/src/components/wallet-header/index.ts

```diff
@@ -0,0 +1,4 @@
+import WalletHeader from './wallet-header';
+import './wallet-header.scss';
```

Comment:
@sergei-deriv I know we are following how it's been done in our codebase, But I would recommend importing the style for a component in the component itself, So even if someone by mistake imports the component directly the component will load its style.

It's just a suggestion, You can ignore it 🙇🏻

Reply:
@sergei-deriv Well at some point we need to start fixing this 😅 Otherwise we will always do the same thing and never improve 😬